dockerfile/README.md

```diff
@@ -1,59 +1,94 @@
-# Dockerfile for WSO2 Enterprise Integrator #
-The Dockerfile defines the resources and instructions to build the Docker images with the WSO2 products and runtime configurations.
+# Dockerfiles for WSO2 Enterprise Integrator #
+This section defines dockerfiles and step-by-step instructions to build docker images for multiple run-times <br>
+provided by WSO2 Enterprise Integrator 6.1.1, namely : <br>
+1. Integrator
+2. Business-process
+3. Broker
+4. Msf4j
```

Comment:
It might be better to write MSF4J in uppercase.

Reply:
Make sense. Will update. Thanks.